Source/WebKit/UIProcess/API/Cocoa/WKFrameInfoPrivate.h

```diff
@@ -43,4 +43,6 @@
 @property (nonatomic, readonly) CGSize _visibleContentSizeExcludingScrollbars WK_API_AVAILABLE(macos(15.4), ios(18.4), visionos(2.4));
 @property (nonatomic, readonly, nullable) SecTrustRef _serverTrust WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA), visionos(WK_XROS_TBA));
 
+- (BOOL)_isSameFrame:(nonnull WKFrameInfo*)frame WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA), visionos(WK_XROS_TBA));
```

Comment:
I think we should use NS_ASSUME_NONNULL_BEGIN/END instead of marking this as non-null.

Also, it would be good to have a test that returns YES and a test that returns NO.